From: Marc Kleine-Budde <mkl@pengutronix.de>
Subject: make busybox kill friendly

don't use "kill -s" which busybox' kill doesn't understand

Signed-off-by: Marc Kleine-Budde <mkl@pengutronix.de>

---
# 20081203: wsa: Fixed in 0.9.4.

 utils/opcontrol |   10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

Index: oprofile-0.9.3/utils/opcontrol
===================================================================
--- oprofile-0.9.3.orig/utils/opcontrol
+++ oprofile-0.9.3/utils/opcontrol
@@ -908,7 +908,7 @@ do_stop()
 		return
 	fi
 
-	kill -s 0 `cat $LOCK_FILE` 2>/dev/null
+	kill -0 `cat $LOCK_FILE` 2>/dev/null
 	if test "$?" -ne 0; then
 		echo "Detected stale lock file. Removing." >&2
 		rm -f "$LOCK_FILE"
@@ -919,7 +919,7 @@ do_stop()
 		echo "Stopping profiling."
 		echo 0 >/dev/oprofile/enable
 	fi
-	kill -s USR2 `cat $LOCK_FILE` 2>/dev/null
+	kill -USR2 `cat $LOCK_FILE` 2>/dev/null
 }
 
 
@@ -932,7 +932,7 @@ do_kill_daemon()
 		return
 	fi
 
-	kill -s 0 `cat $LOCK_FILE` 2>/dev/null
+	kill -0 `cat $LOCK_FILE` 2>/dev/null
 	if test "$?" -ne 0; then
 		echo "Detected stale lock file. Removing." >&2
 		rm -f "$LOCK_FILE"
@@ -1274,7 +1274,7 @@ do_start_daemon()
 {
  
 	if test -f "$LOCK_FILE"; then
-		kill -s 0 `cat $LOCK_FILE` 2>/dev/null
+		kill -0 `cat $LOCK_FILE` 2>/dev/null
 		if test "$?" -eq 0; then
 			return;
 		else
@@ -1341,7 +1341,7 @@ do_start()
 	if test "$KERNEL_SUPPORT" = "yes"; then
 		echo 1 >$MOUNT/enable
 	fi
-	kill -s USR1 `cat $LOCK_FILE` 2>/dev/null
+	kill -USR1 `cat $LOCK_FILE` 2>/dev/null
 	echo "Profiler running."
 }
 
